index.js

```diff
@@ -102,3 +103,14 @@ function create() {
     $(".new-contact").fadeOut()
   })
 }
+
+function moreInfo() {
+  $(".name").on("click", function() {
+    $(this).toggleClass("display")
```

Comment:
I _think_ jQuery made a shortcut for this functionality. I think you can just do `$(this).toggle()`.